build/mac/get_name.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# intel or arm
+ARCHITECHTURE=$1
+# Get version from package.json, get line with version on it, then remove everything but version
+# and replace . with _
+VERSION=$(cat package.json | grep 'version":' | sed 's/[^0-9.]//g' | sed 's/[.]/_/g')
+# Commit {DAY}{MONTH}_{HOUR}{SECOND}
```

Comment:
I thought this would be very useful in the name, since multiple versions can be built (i.e. with translations or rc versions)